suggestion.go

```diff
@@ -79,6 +81,14 @@ func addGreeting(r *extypes.Response, pkg, student string) {
 	}
 }
 
+func addTip(r *extypes.Response) {
+	if r.LenImprovements() < 3 {
+		rand.Seed(time.Now().UnixNano())
+		t := rand.Perm(len(gtpl.Tips))[0]
```

Comment:
Maybe just use `rand.Intn(len(gtpl.Tips))`?

Reply:
You're quite right–and I just mentored a bunch of Robot Name exercises! 😂